CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change history for stripes-data-transfer-components
 
-## 1.1.0 (IN PROGRESS)
+## 2.1.0 (IN PROGRESS)
+
+## [2.0.0](https://github.com/folio-org/stripes-data-transfer-components/tree/v2.0.0) (2020-12-06)
```

Comment:
Please, add link to the full change log:
`[Full Changelog](https://github.com/folio-org/stripes-data-transfer-components/tree/v1.0.1...v2.0.0)`